docs/user-guide/services-firewalls.md

```diff
@@ -6,6 +6,51 @@ exposure to the internet.  When exposing a service to the external world, you ma
 one or more ports in these firewalls to serve traffic.  This document describes this process, as
 well as any provider specific details that may be necessary.
 
+### Restrict Access For LoadBlancer Service
+
+ When using a Service with `spec.type: LoadBalancer`, you can specify the IP ranges that are allowed to access the load balancer
+ by using `spec.loadBalancerSourceRanges`. This field takes a list of IP CIDR ranges, which Kubernetes will use to configure firewall exceptions.
+ This feature is currently supported on Google Compute Engine, Google Container Engine and AWS. This field will be ignored if the cloud provider does not support the feature.
+
+ In the following example, a load blancer will be created that is only accessible to clients with IP addresses from 10.180.0.0/16 and 10.245.0.0/24.
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: myapp
+spec:
+  ports:
+    - port: 8765
+      targetPort: 9376
+  selector:
+    app: example
+  type: LoadBalancer
+  loadBalancerSourceRanges:
+  - 10.180.0.0/16
+  - 10.245.0.0/24
+```
+
+ In the following example, a load blancer will be created that is only accessible to clients with IP addresses from 10.180.0.10, 10.245.0.10 and 1.2.3.4.
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: myapp
+spec:
+  ports:
+    - port: 8765
+      targetPort: 9376
+  selector:
+    app: example
+  type: LoadBalancer
+  loadBalancerSourceRanges:
+  - 10.180.0.10/32
```

Comment:
At the beginngin of the doc you have:

" When using a Service with `spec.type: LoadBalancer`, you can specify the IP ranges that are allowed to access the load balancer"

on gce though, will actually specifying 10.180.0.10/32 mean that curling the loadbalancer public ip will not work from 10.180.0.10? I'm guessing it'll still work, because when we access a public ip, the packet is natted to *our* public ip. So in that case, the source range is only applied to traffic to that vm through some other source (not the loadbalancer). right?